app/core/store/modules/campaigns.js

```diff
@@ -4,20 +4,23 @@ export default {
   namespaced: true,
   state: {
     byId: {},
-    bySlug: {}
+    bySlug: {},
+    latestCampaignId: null
   },
 
   mutations: {
     setCampaignData: (state, campaignData) => {
       Vue.set(state.byId, campaignData._id, campaignData)
       Vue.set(state.bySlug, campaignData.slug, campaignData)
+      state.latestCampaignId = campaignData._id
     }
   },
 
   getters: {
     getCampaignData: (state) => (idOrSlug) => {
       return state.byId[idOrSlug] || state.bySlug[idOrSlug]
-    }
+    },
+    getLatestCampaignId: (state) => state.latestCampaignId
```

Comment:
What is meant by "latest" campaign id?  Is it different than "current"?  Latest gives the impression that it's expected to rotate or update frequently.

Reply:
Yeah. Excellent naming comment. Will update to `getCurrentCampaignId`.